ports-wip/cronie/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This code Copyright 2012 Todd Mortimer <todd.mortimer@gmail.com>
+#
+# You may do whatever you like with this code, provided the above
+# copyright notice and this paragraph are preserved.
+
+
+set -e
+source ../../lib.sh
+TASK=fetch
+
+DISTVER="cronie-1.5.2"
```

Comment:
@berryamin I think we can delete cronie from `ports-wip` because it's already in `ports`